.github/workflows/release-workflow.yaml

```diff
@@ -0,0 +1,73 @@
+name: Release new version
+
+on:
+  workflow_call:
+    inputs:
+      majorRelease:
+        required: false
+        type: string
+        description: 'Release as version. Example values are: minor, major, 5.6.1, 1.0.0. Leave empty for regular release.'
+      branch:
+        required: true
+        type: string
+        description: 'main or 0.x branch'
+
+jobs:
+  # integration-tests:
```

Comment:
Shoud we keep that commented ?

Reply:
It was commented beforehand, not sure why. I'll check if integration tests pass on main and if so uncomment it.

Reply:
It fails on main so I'll keep this as is for now.